test/nstemplatetier/assertion.go

```diff
@@ -1,72 +0,0 @@
-package nstemplatetier
-
-import (
-	"context"
-
-	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
-	"github.com/codeready-toolchain/toolchain-common/pkg/test"
-
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	"k8s.io/apimachinery/pkg/types"
-	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
-)
-
-// Assertion an assertion helper for an NSTemplateTier
-type Assertion struct {
-	tier           *toolchainv1alpha1.NSTemplateTier
-	client         runtimeclient.Client
-	namespacedName types.NamespacedName
-	t              test.T
-}
-
-func (a *Assertion) loadResource() error {
-	tier := &toolchainv1alpha1.NSTemplateTier{}
-	err := a.client.Get(context.TODO(), a.namespacedName, tier)
-	a.tier = tier
-	return err
-}
-
-// AssertThatNSTemplateTier helper func to begin with the assertions on an NSTemplateTier
-func AssertThatNSTemplateTier(t test.T, name string, client runtimeclient.Client) *Assertion {
-	return &Assertion{
-		client:         client,
-		namespacedName: test.NamespacedName(test.HostOperatorNs, name),
-		t:              t,
-	}
-}
```

Comment:
I know that the custom assertion doesn't make much sense when the status-related checks are removed, but keep in mind that the assertion is used & extended by the @mfrancisc's PR, so it would be fine to keep at least these three functions to minimize the conflicts in his PR

Reply:
yes thanks for this,  I did try to keep them, but the linter complained, so removed it , but can bring it back

Reply:
@MatousJobanek thanks for the heads up! 

@fbm3307 In case this is causing you issues, feel free to drop it. I can add it back , in case you merge this first 👍

Reply:
@MatousJobanek  looks like the required golang ci test case failed here, i guess we shouldn't force merge ?@mfrancisc can add it in his PR ? WDYT?